### PR TITLE
Added IPv6 information for Kubelet arguments

### PR DESCRIPTION
**Pull request by mmueen**
in docs\reference\generated\kubelet.md , IPv6 information was added for the following arguments.

 --address ip 
 --healthz-bind-address ip

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6498)

### CODE REVIEW

**Comment by mmueen** [comment]
cc: @leblancd @rpothier @pmichali

**Comment by zacharysarah** [comment]
@mmueen :wave: This looks related to #6465, so I'll change the base to `release-1.9` and add tracking info for it. If I'm wrong, please let me know and I'll change it back! 😸

**Comment by steveperry-53** [comment]
The files in this PR are auto generated. Please close this PR. I will open a PR that updates the auto generated docs.

**Comment by steveperry-53** [comment]
The files in this PR are auto generated. So any changed here will be overwritten when we update the auto generated docs.

**Comment by zacharysarah** [comment]
@steveperry-53 Good catch. Where should these updates be performed?

File: docs/reference/generated/kube-proxy.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ kube-proxy
 
 ```
       --azure-container-registry-config string       Path to the file container Azure container registry configuration information.
-      --bind-address ip                              The IP address for the proxy server to serve on (set to 0.0.0.0 for all interfaces) (default 0.0.0.0)
+      --bind-address ip                              The IP address for the proxy server to serve on (set to 0.0.0.0 for all IPv4 interfaces and :: for all IPv6 interfaces) (default 0.0.0.0)
```
Comment (tengqm) [inline docs/reference/generated/kube-proxy.md]: should be fixed here: https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-proxy/app/server.go#L128

File: docs/reference/generated/kube-proxy.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ SupportIPVSProxyMode=true|false (ALPHA - default=false)
 TaintBasedEvictions=true|false (ALPHA - default=false)
 TaintNodesByCondition=true|false (ALPHA - default=false)
       --google-json-key string                       The Google Cloud Platform Service Account JSON Key to use for authentication.
-      --healthz-bind-address ip                      The IP address and port for the health check server to serve on (set to 0.0.0.0 for all interfaces) (default 0.0.0.0:10256)
+      --healthz-bind-address ip                      The IP address and port for the health check server to serve on (set to 0.0.0.0 for all IPv4 interfaces and :: for all IPv6 interfaces) (default 0.0.0.0:10256)
```
Comment (tengqm) [inline docs/reference/generated/kube-proxy.md]: should be fixed here: https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-proxy/app/server.go#L131

File: docs/reference/generated/kube-proxy.md [hunk 3]
```diff
@@ -80,7 +80,7 @@ TaintNodesByCondition=true|false (ALPHA - default=false)
       --kubeconfig string                            Path to kubeconfig file with authorization information (the master location is set by the master flag).
       --masquerade-all                               If using the pure iptables proxy, SNAT all traffic sent via Service cluster IPs (this not commonly needed)
       --master string                                The address of the Kubernetes API server (overrides any value in kubeconfig)
-      --metrics-bind-address ip                      The IP address and port for the metrics server to serve on (set to 0.0.0.0 for all interfaces) (default 127.0.0.1:10249)
+      --metrics-bind-address ip                      The IP address and port for the metrics server to serve on (set to 0.0.0.0 for all IPv4 interfaces and :: for all IPv6 interfaces) (default 127.0.0.1:10249)
```
Comment (tengqm) [inline docs/reference/generated/kube-proxy.md]: should be fixed here: https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-proxy/app/server.go#L132

File: docs/reference/generated/kubelet.md [hunk 4]
```diff
@@ -36,7 +36,7 @@ kubelet
 ### Options
 
 ```
-      --address ip                                               The IP address for the Kubelet to serve on (set to 0.0.0.0 for all interfaces) (default 0.0.0.0)
+      --address ip                                               The IP address for the Kubelet to serve on (set to 0.0.0.0 for all IPv4 interfaces and :: for all IPv6 interfaces) (default 0.0.0.0)
```
Comment (tengqm) [inline docs/reference/generated/kubelet.md]: should be fixed upstream here: https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/options/options.go#L385

File: docs/reference/generated/kubelet.md [hunk 5]
```diff
@@ -124,7 +124,7 @@ TaintNodesByCondition=true|false (ALPHA - default=false)
       --file-check-frequency duration                            Duration between checking config files for new data (default 20s)
       --google-json-key string                                   The Google Cloud Platform Service Account JSON Key to use for authentication.
       --hairpin-mode string                                      How should the kubelet setup hairpin NAT. This allows endpoints of a Service to loadbalance back to themselves if they should try to access their own Service. Valid values are "promiscuous-bridge", "hairpin-veth" and "none". (default "promiscuous-bridge")
-      --healthz-bind-address ip                                  The IP address for the healthz server to serve on. (set to 0.0.0.0 for all interfaces) (default 127.0.0.1)
+      --healthz-bind-address ip                                  The IP address for the healthz server to serve on. (set to 0.0.0.0 for all IPv4 interfaces and :: for all IPv6 interfaces) (default 127.0.0.1)
```
Comment (tengqm) [inline docs/reference/generated/kubelet.md]: should be fixed upstream here: https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/options/options.go#L430

**Pull request merged**